### DISCUSSION TITLE
"Please sign in to confirm you are not a bot" - Youtube Issue

### DISCUSSION THREAD

**Discussion by moerk824** [jagrosh/MusicBot#1684] [Troubleshooting]
Hey, I am hosting the Jagrosh Music Bot using a server hosting platform. As soon as I want to play a song it instantly leaves and says "Please sign in to confirm you are not  a bot". Can someone please help me fixing that issue. I heard  something about vps, but I dont know how to do that, when using a server hosting platform. 
Thank you

**Comment by vincedevriesdev** [comment]
i also have this issue yea. 
have not found a solution.

**Comment by T-J-DEV** [comment]
As of today i have this same issue. no fix found either.

**Comment by Detrailia** [comment]
I believe bot needs to be updated!!! Hopefully its soon

**Comment by MichailiK** [comment]
There is already an issue for this:

- #1588
